sdk/highlight-py/highlight_io/sdk.py

```diff
@@ -21,6 +21,11 @@
 from opentelemetry.trace import INVALID_SPAN
 
 from highlight_io.integrations import Integration
+from highlight_io.utils.lru_cache import LRUCache
+from highlight_io.integrations.requests import RequestsIntegration
+
+
+DEFAULT_INTEGRATIONS = [[RequestsIntegration.INTEGRATION_KEY, RequestsIntegration]]
```

Comment:
make INTEGRATION_KEY an attribute of the `Integration` class and have every integration override it